material-components/material-components-android-examples
# Reply elevation color

# Summary
- Updated the application to support the "inverse" color scheme. (Higher elevation = lighter color, lower elevation = darker)
- Made tweaks to the star IconButton logic which includes: ToggleDrawable and Disabiling Elevation Overlay
- Updated the API 31 < devices to use a new color scheme based off of: `FAAB1A`
- About the Star Icon: Design wants us to use the checked state with `colorOnSurface` and default state: `colorOnSurfaceVariant`.

# In-depth Notes
- Binding Adapter is used for updating the background colors
- Many files regarding bottomnavdrawer can be deleted, however need guidance on which ones will not hamper the functionality of the application
- Elevation Overlay disabled for the Star Button is due to how material elevation works with absolute elevation.

# Test
1. Pull Branch
2. Run the Application
3. Make sure it mimics the screenshots

# Screenshots
## API 31
Home Screen      |  Email Screen
:-------------------------:|:-------------------------:
<img src="https://user-images.githubusercontent.com/32623283/139758130-86a51979-2686-4986-a8ff-f6842e83fa9b.png" width="300">| <img src="https://user-images.githubusercontent.com/32623283/139758080-45fba204-71e6-44da-bc38-d0525e4b1e28.png" width="300">|


## < API 31 (API 28)
Home Screen           |  Email Screen
:-------------------------:|:-------------------------:
<img src="https://user-images.githubusercontent.com/32623283/139758183-d90bb2bd-05bd-4473-80fe-4b1672553894.png" width="300">| <img src="https://user-images.githubusercontent.com/32623283/139757946-e773a05b-57aa-41b9-8203-31b3dea49132.png" width="300">
